.gitignore

```diff
@@ -139,3 +139,10 @@ checkpoints/
 experiments/
 wandb/
 raylib/
+
+c_gae.c
```

Comment:
These files are all generated when you run `pip install --editable .` -- seems like they should be in the .gitignore